corehq/apps/cloudcare/static/cloudcare/js/form_entry/form_ui.js

```diff
@@ -290,7 +292,19 @@ hqDefine("cloudcare/js/form_entry/form_ui", [
                     return options.target;
                 },
                 key: function (data) {
-                    return ko.utils.unwrapObservable(data.uuid) || ko.utils.unwrapObservable(data.ix);
+                    let key = ko.utils.unwrapObservable(data.uuid);
+                    if (key) {
+                        return key;
+                    }
+                    key = ko.utils.unwrapObservable(data.ix);
+
+                    const isRepeatDummy = data.type === constants.GROUP_TYPE && data.exists === "false";
```

Comment:
I'm guessing this isn't fixable within the scope of this PR, but seeing `=== "false"` is jarring, seems like `data` ought to be getting parsed as JSON somewhere upstream.

Reply:
I agree. Formplayer sends it as a string not a boolean. He said it already existed like that and was hesitant to change it.